src/app/protocol.ts

```diff
@@ -391,7 +344,7 @@ class XpraProtocol {
       }
     } else {
       //decode raw packet string into objects:
-      let packet = null;
+      let packet: any;
```

Comment:
Shouldn't we define the object 'xprapacket' instead of any?

Reply:
Yes, though throughout the code if I'm not confident on the correct type I'm leaving it as any so as to not cause conflicts if I can't confidently infer the correct type for an object